libres/lib/analysis/std_enkf.cpp

```diff
@@ -75,8 +76,9 @@ struct std_enkf_data_struct {
     double truncation; // Controlled by config key: ENKF_TRUNCATION_KEY
     int subspace_dimension; // Controlled by config key: ENKF_NCOMP_KEY (-1: use Truncation instead)
     long option_flags;
-    bool use_EE;
-    bool use_GE;
+    bool __use_EE;
```

Comment:
Perhaps add a comment to here that states that both `__use_EE` and `__use_GE` are deprecated and shall be removed.
I don't have a clear overview as to what has to be done in order for us to be able to remove them.
Is this something that can be done as part of the ies-refactoring you are doing now, or do we have to make bigger changes?

Reply:
> Perhaps add a comment to here that states that both __use_EE and __use_GE are deprecated and shall be removed.

I will.

> I don't have a clear overview as to what has to be done in order for us to be able to remove them.

The *only* thing which still keeps them around is the users current ability to set them in the config file - i.e. this works:
```
ANALYSIS_SET STD_ENKF USE_EE True
ANALYSIS_SET STD_ENKF USE_GE False
```
I assume the actual use of such settings is very limited; so after some brief investigation (log peeking?) I guess they can be removed.

> Is this something that can be done as part of the ies-refactoring you are doing now

Absolutely - as explained it is more of a political than a technical question.

Reply:
Good stuff - I've created and issue here: https://github.com/equinor/ert/issues/2534